build-system/tasks/prepend-global/index.js

```diff
@@ -155,20 +189,47 @@ async function applyConfig(
   if (opt_derandomize) {
     configJson = derandomize_(target, configJson);
   }
-  const fileString = await prependConfig(
-    JSON.stringify(configJson),
-    targetString
-  );
-  sanityCheck(fileString);
-  await writeTarget_(target, fileString, argv.dryrun);
+
   const details =
     '(' +
-    cyan(config) +
+    cyan(type) +
     (opt_localDev ? ', ' + cyan('localDev') : '') +
     (opt_fortesting ? ', ' + cyan('test') : '') +
     (opt_derandomize ? ', ' + cyan('derandomized') : '') +
     ')';
-  log('Applied AMP config', details, 'to', cyan(path.basename(target)));
+  log('Created AMP config', details, 'to', cyan(path.basename(target)));
```

Comment:
Changed `Created` to `Applying` and removed `path.basename()` so the extension gets printed.

```suggestion
  log('Applying AMP config', details, 'to', cyan(target));
```